pkg/action/executor/resize_container.go

```diff
@@ -372,7 +374,7 @@ func (r *ContainerResizer) preActionCheck(resizeSpec *containerResizeSpec, pod *
 	fullName := fmt.Sprintf("%s/%s", pod.Namespace, pod.Name)
 
 	// Check if the pod privilege is supported
-	if !util.SupportPrivilegePod(pod) {
+	if !util.SupportPrivilegePod(pod, r.sccAllowedSet) {
 		err := fmt.Errorf("The pod %s has privilege requirement unsupported", fullName)
```

Comment:
The log should be something like "Target pod has unsupported SCC, please add the SCC to --scc-support argument in kubeturbo yaml to allow execution."